python/binding/tritonserver_pybind.cc

```diff
@@ -0,0 +1,1962 @@
+          &PyServerOptions::SetBufferManagerThreadCount)
+      .def(
+          "set_model_load_thread_count",
+          &PyServerOptions::SetModelLoadThreadCount)
+      .def("set_model_namespacing", &PyServerOptions::SetModelNamespacing)
+      .def("set_log_file", &PyServerOptions::SetLogFile)
+      .def("set_log_info", &PyServerOptions::SetLogInfo)
+      .def("set_log_warn", &PyServerOptions::SetLogWarn)
+      .def("set_log_error", &PyServerOptions::SetLogError)
+      .def("set_log_format", &PyServerOptions::SetLogFormat)
+      .def("set_log_verbose", &PyServerOptions::SetLogVerbose)
+      .def("set_metrics", &PyServerOptions::SetMetrics)
+      .def("set_gpu_metrics", &PyServerOptions::SetGpuMetrics)
+      .def("set_cpu_metrics", &PyServerOptions::SetCpuMetrics)
+      .def("set_metrics_interval", &PyServerOptions::SetMetricsInterval)
+      .def("set_backend_directory", &PyServerOptions::SetBackendDirectory)
+      .def("set_repo_agent_directory", &PyServerOptions::SetRepoAgentDirectory)
+      .def(
+          "set_model_load_device_limit",
+          &PyServerOptions::SetModelLoadDeviceLimit)
+      .def("set_backend_config", &PyServerOptions::SetBackendConfig)
+      .def("set_host_policy", &PyServerOptions::SetHostPolicy)
+      .def("set_metrics_config", &PyServerOptions::SetMetricsConfig);
+
+  // TRITONSERVER_Server
+  py::class_<PyServer>(m, "TRITONSERVER_Server")
+      .def(py::init<PyServerOptions&>())
+      .def("stop", &PyServer::Stop)
+      .def("register_model_repository", &PyServer::RegisterModelRepository)
+      .def("unregister_model_repository", &PyServer::UnregisterModelRepository)
+      .def("poll_model_repository", &PyServer::PollModelRepository)
+      .def("poll_model_repository", &PyServer::PollModelRepository)
+      .def("is_live", &PyServer::IsLive)
+      .def("is_ready", &PyServer::IsReady)
+      .def("model_is_ready", &PyServer::ModelIsReady)
+      .def("model_batch_properties", &PyServer::ModelBatchProperties)
+      .def(
+          "model_transaction_properties", &PyServer::ModelTransactionProperties)
+      .def("metadata", &PyServer::Metadata)
+      .def("model_metadata", &PyServer::ModelMetadata)
+      .def("model_statistics", &PyServer::ModelStatistics)
+      .def("model_config", &PyServer::ModelConfig)
+      .def("model_index", &PyServer::ModelIndex)
+      .def("load_model", &PyServer::LoadModel)
+      .def("load_model_with_parameters", &PyServer::LoadModelWithParameters)
+      .def("unload_model", &PyServer::UnloadModel)
+      .def("unload_model_and_dependents", &PyServer::UnloadModelAndDependents)
+      .def("metrics", &PyServer::Metrics)
+      .def("infer_async", &PyServer::InferAsync);
+
+  // TRITONSERVER_MetricFamily
+  py::class_<PyMetricFamily>(m, "TRITONSERVER_MetricFamily")
+      .def(py::init<
+           TRITONSERVER_MetricKind, const std::string&, const std::string&>());
+
+  // TRITONSERVER_Metric
+  py::class_<PyMetric>(m, "TRITONSERVER_Metric")
```

Comment:
Perhaps remove the "TRITONSERVER_" prefix from the python bindings names?

Reply:
I would like to keep the name align with the C API, but I can change it if the majority prefer a concise naming

Reply:
I agree keeping the naming aligned is useful and helps map them. 

One option could be to have the module name be triton_bindings.TRITONSERVER. <suffix>